src/main/kotlin/org/koitharu/kotatsu/parsers/model/Manga.kt

```diff
@@ -67,7 +67,14 @@ class Manga(
 	 * Manga source
 	 */
 	@JvmField val source: MangaSource,
-) {
+
+	@JvmField val date: Long?  = null,
```

Comment:
Please, do not add new fields into core models (manga, chapter, page, etc.)

Even if it might be useful, modification of these classes will lead to a lot of changes on the app side and all fields should be supported by (almost) all sources

Reply:
ok, fixed.